sdk/cosmos/azure-cosmos-spark_3-1_2-12/README.md

```diff
@@ -29,6 +29,7 @@ https://github.com/Azure/azure-sdk-for-java/issues/new
 #### azure-cosmos-spark_3-1_2-12
 | Connector    | Supported Spark Versions | Minimum Java Version | Supported Scala Versions | Supported Databricks Runtimes |
 |--------------| ------------------------ | -------------------- | -----------------------  | ----------------------------- |
+| 4.19.0       | 3.1.1 - 3.1.2            | 8                    | 2.12                     | 8.\*, 9.\*                    |
```

Comment:
also update the spark connector version for other versions in the same file 
and also update the spark version in the download section in the same file 

also need to update the spark version in the 3_2 quick start file

Reply:
thanks Annie!!